### PR TITLE
display multiple notices at once

### PR DESCRIPTION
**Pull request by nathanstitt**
screen shot 2017-01-03 at 1 05 27 pm

### CODE REVIEW

File: coach/src/navigation/index.cjsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ Navigation = React.createClass
 
   onCourseEnded: (notice) ->
     NotificationActions.acknowledge(notice)
-    channel.emit('show.registration', view: 'second-semester-registration')
+    channel.emit('show.second-semester-registration', view: 'second-semester-registration')
```
Comment (nathanstitt) [inline coach/src/navigation/index.cjsx]: this is an unrelated bug fix for the clicking on the second semester link.  I had it as a separate pr but it snuck in here

File: shared/src/model/notifications.coffee [hunk 2]
```diff
@@ -56,11 +56,10 @@ Notifications = {
       @emit('change')
 
   getActive: ->
-    return NOTICES[0] if NOTICES.length
+    active = clone(NOTICES)
```
Comment (Dantemss) [inline shared/src/model/notifications.coffee]: Hm I'm a bit confused by the interaction between `NOTICES`, `display` and `getActive`.

So `NOTICES` seems to contain an array of the things that are actually displayed and is set by the `display` method. `display` is called from `setCourseRole`, which means when you pick a dashboard?

`getActive` includes all the `NOTICES` but also adds extra notices from the pollers... (how are these different?)

I think we might want to document what's happening here for the future.

Comment (Dantemss) [inline shared/src/model/notifications.coffee]: Ah I think I see now... `NOTICES` are set by admins... the poller stuff handles the automated notices... we might want to add a comment explaining this.
